dot/types/account.go

```diff
@@ -13,9 +13,9 @@ type AccountInfo struct {
 	RefCount uint32
 	// The additional data that belongs to this account. Used to store the balance(s) in a lot of chains.
 	Data struct {
-		Free       common.Uint128
-		Reserved   common.Uint128
-		MiscFrozen common.Uint128
-		FreeFrozen common.Uint128
+		Free       *scale.Uint128
+		Reserved   *scale.Uint128
+		MiscFrozen *scale.Uint128
+		FreeFrozen *scale.Uint128
```

Comment:
are pointers optionals in the new scale? should this be `scale.Uint128` or does scale handle this as a non-option?

Reply:
`**scale.Uint128` is optional.  Same for `**big.Int`.